pkg/storage/system_intake.go

```diff
@@ -418,6 +424,7 @@ func (s *Store) FetchIntakesForAdmins(ctx context.Context) ([]*models.SystemInta
 }
 
 // FetchSystemIntakesByStatuses queries the DB for all system intakes matching a status filter
+// Is this in use? It should be removed post admin actions v2 launch or modified
```

Comment:
To my knowledge, this is what the admin home currently uses to populate the `open` and `closed` tabs.

![image](https://github.com/CMSgov/easi-app/assets/15203744/45a0b158-b442-467d-818a-42c086a10ec9)

We will probably need to modify/clean this up to no longer use `status` when we deprecate statuses all-together

Reply:
Ah, to clarify my comment, the above function isn't in use and the one you're addressing is used instead. So I think we could probably use the unused one and replace this one, potentially. But we can revisit it in the cleanup.

Reply:
@mynar7 I ended up realizing that we needed to clean up this function sooner rather than later, since it's using V1 status which isn't set by V2 actions

https://github.com/CMSgov/easi-app/pull/2318/commits/14cbc35584768333769ad3fba732f2698e034d3a

I put this commit on @adamodd's PR which also updated the admin home page